nservicebus/scalability-and-ha/scale-out.md

```diff
@@ -55,6 +55,7 @@ snippet:Routing-StaticRoutes-Endpoint-Msmq
 
 NOTE: System administrators are able to spin-up new instances of the endpoint should the load increase and the only requirement is adding an entry to the routing file. No changes in the source code are required.
 
+WARNING: When using this scaling out technique in a mixed version environment make sure to deploy a distributor in front of the scaled out V6 endpoint if that endpoint needs to subscribe to events published by endpoints using versions lower than 6. Please refer to the V6 version of [the distributor sample](/samples/scaleout/distributor/) for details on how to do it. Otherwise the subscribed events will be delivered to every instance of the scaled out endpoint (instead of being delivered to a single instance using a load-balancing algorithm).
```

Comment:
@indualagarsamy V6 publisher is not relevant here because it does not change any behavior.